mono/metadata/domain-internals.h

```diff
@@ -608,7 +608,12 @@ MonoJitInfo* mono_jit_info_table_find_internal (MonoDomain *domain, char *addr,
 void mono_enable_debug_domain_unload (gboolean enable);
 
 MonoReflectionAssembly *
-mono_domain_try_type_resolve_checked (MonoDomain *domain, char *name, MonoObject *tb, MonoError *error);
+mono_domain_try_type_resolve_name (MonoDomain *domain, const char *name, MonoError *error);
+
+typedef struct _MonoReflectionTypeBuilder MonoReflectionTypeBuilder;
```

Comment:
This is unnecessary.

Reply:
Not entirely..but I'll refine it. Apparently the ancient Visual C++ behavior is officially new in C11 or C++11 and I'm seeing warnings. It is entirely unnecessary if we traffic in MonoObject* instead of MonoReflectionTypeBuilder* granted, but I'd rather pass the more specific type. Anyway let me back to this.